pkg/scheduler/objects/queue.go

```diff
@@ -131,15 +131,17 @@ func NewConfiguredQueue(conf configs.QueueConfig, parent *Queue) (*Queue, error)
 	// add to the parent, we might have an overall lock already
 	// still need to make sure we lock the parent so we do not interfere with scheduling
 	if parent != nil {
+		sq.mergeProperties(parent.getProperties(), conf.Properties)
+		sq.UpdateQueueProperties()
 		sq.QueuePath = parent.QueuePath + configs.DOT + sq.Name
 		err := parent.addChildQueue(sq)
 		if err != nil {
 			return nil, fmt.Errorf("configured queue creation failed: %w", err)
 		}
 		// pull the properties from the parent that should be set on the child
```

Comment:
Nit: This comment should probably be moved to just before sq.mergeProperties above.